kivy/app.py

```diff
@@ -383,8 +383,13 @@ def on_resume(self):
 
     from kivy.app import async_runTouchApp
     from kivy.uix.label import Label
-
-    trio.run(async_runTouchApp, Label(text='Hello, World!'), async_lib='trio')
+    
+    from functools import partial
+    
+    # use functools.partial() to pass keyword arguments:
+    async_runTouchApp_func = partial(async_runTouchApp, async_lib='trio')
+    
+    trio.run(async_runTouchApp_func, Label(text='Hello, World!'))
```

Comment:
```suggestion

    from functools import partial

    # use functools.partial() to pass keyword arguments:
    async_runTouchApp_func = partial(async_runTouchApp, async_lib='trio')

    trio.run(async_runTouchApp_func, Label(text='Hello, World!'))
```

I thought the PEP8 issue was the line was too long, but it's actually that you had whitespace on empty lines.